lib/facility_builder.rb

```diff
@@ -0,0 +1,92 @@
+class FacilityBuilder
+
+  attr_reader :co_facilities, :ny_facilities, :mo_facilities
+  def initialize
+    @co_facilities = []
+    @ny_facilities = []
+    @mo_facilities = []
+  end
+
+
+  def create_co_facilities(data)
+
+    data.each do |data|
+      data.default = "not available"
+      new_facility = Facility.new({
+        name: data[:dmv_office],
+        address: "#{data[:address_li]} #{data[:address__1]} #{data[:city]} #{data[:state]} #{data[:zip]}",
+        phone: data[:phone]
+      })
+
+      if data[:services_p].include?('registration')
+
+        new_facility.add_service("Vehicle Registration")
+      end
+
+      if data[:services_p].include?("renewals")
+        new_facility.add_service("Renew Drivers License")
+      end
+
+      @co_facilities << new_facility
+
+    end
+    return @co_facilities
+  end
+
+  def create_ny_facilities(data)
+
+    @ny_facilities = data.map do |data|
```

Comment:
You used a .each for co, but a .map for ny. Probably just the .each would work for both since you're storing them in the instance variable